components/portal/messages/partials/notifications-list-item.html

```diff
@@ -37,7 +37,9 @@
     <md-button class="md-raised md-accent"
                ng-href="{{ notification.actionButton.url }}"
                ng-if="notification.actionButton && notification.actionButton.url"
-               ng-click="pushGAEvent('notifications page', 'take action', notification.actionButton.url)">
+               ng-click="pushGAEvent('notifications page', 'take action', notification.actionButton.url)"
+               target="_blank"
```

Comment:
iirc we've got existing code somewhere for detecting whether a URL represents an "external" or "internal" link. Could we use that here so that the open-in-new-tab-ness is predicated on the external-ness?

Reply:
Discussed in stand-up -- no existing code to determine this info.